sdcm/nemesis.py

```diff
@@ -139,6 +139,9 @@ class Nemesis:  # pylint: disable=too-many-instance-attributes,too-many-public-m
     disruptions_list = []
     has_steady_run = False
     DISRUPT_NAME_PREF = "disrupt_"
+    topology_changes = False
```

Comment:
Let's start adding description to the flag, so that you could understand what it represents.
Also please accumulate flags in one place and add : bool to them

Reply:
done